### PR TITLE
fix Learn More link and added Report Issue

### PR DESCRIPTION
**Pull request by todrobbins**
MENUS ⚡️

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -45,8 +45,12 @@ const template = [
     submenu: [
       {
         label: 'Learn More',
-        click () { require('electron').shell.openExternal('https://cabal.chat') }
-      }
+        click () { require('electron').shell.openExternal('https://cabal-club.github.io/') }
```
Comment (nikolaiwarner) [inline index.js]: Does this need to change?

Comment (todrobbins) [reply index.js]: Hmm, looks like an SSL issue now that I look at it closer. Trying to load https://cabal.chat just stalls in Chrome, but simply `cabal.chat` redirects just fine to https://cabal-club.github.io/

I'll patch this then.

Comment (todrobbins) [reply index.js]: resolved with 2700c3d

**Comment by todrobbins** [comment]
@nikolaiwarner made the link fix/revert.

**Pull request merged**